.gitignore

```diff
@@ -3,5 +3,7 @@ target/
 user/
 figures/
 vizlab/
-makefile
+[Mm]akefile
 .Rproj.user
+.Rhistory
+internal-analytics.Rproj
```

Comment:
so as we're learning, if we don't ignore internal-analytics.Rproj, we might have a better time unifying everyone's tab/space issue....